packages/new-polymath-ui/src/components/Table/Rows/Row/index.tsx

```diff
@@ -12,15 +12,15 @@ export const Row: FC<Props> = ({ row, small }) => {
     <sc.Row
       {...row.getRowProps()}
       selected={row.isSelected}
-      hasError={!row.isValid}
+      hasError={row.isValid === false}
```

Comment:
Why was this change required?

Reply:
Because if `isValid` is not defined it shouldn't count has being `false`. The `isValid` property can or not be defined based on what "feature" is activated in the `Table` (here it's the `useCsvParser` feature).

Reply:
@Grsmto but `undefined` is a falsy value. Therefore you can just do `!row.isValid` which by javascript's type coercion will become a boolean

Reply:
Oooh, wait so we have 3 states? valid, invalid and neither?